app/controllers/admin/people_controller.rb

```diff
@@ -0,0 +1,7 @@
+module Admin
```

Comment:
I think the markup should be included in this commit. Without the markup, what is this commit doing?